src/tools/clippy/tests/ui/if_let_some_result.fixed

```diff
@@ -20,8 +20,8 @@ fn strange_some_no_else(x: &str) -> i32 {
     }
 }
 
-fn main() {
-    let _ = str_to_int("1");
-    let _ = str_to_int_ok("2");
-    let _ = strange_some_no_else("3");
```

Comment:
Rust allows `unused`-lints in normal compiler UI tests. Are they now also disabled in Clippy's UI tests? (When running them locally via `cargo test` or inside the Clippy CI)

Reply:
Well, looks like @rust-log-analyzer answered the question for us ^^. We could add something like that in the future to Clippy as well ^^

Reply:
> Rust allows `unused`-lints in normal compiler UI tests.

Didn't know that. Sounds good for Clippy.